### ISSUE TITLE
Bug: Opening dropdown menu via keyboard

### ISSUE THREAD

**Issue by SaiShan** [SAP/fundamental-styles#83]
#### Is this a bug, enhancement, or feature request?
Bug

#### Briefly describe your proposal.

Opening the dropdown menu via ALT + DownArrow. 
      - Combobox 
      - Dropdown
      - Localization editor
      - Mega menu
      - Popover
      - Search input
      - Shell bar
      - Tree

**Comment by droshev** [comment]
most of the components have been refactored and the keyboard support should be done in ngx or react
